scripts/serverless.js

```diff
@@ -36,6 +36,13 @@ if (nodeVersion < 10) {
   );
 }
 
+if (/slss$/i.test(process.argv[1])) {
```

Comment:
Let's add it here: https://github.com/serverless/serverless/blob/c6e25bfc2008ec492a212574972feb6bfcf2f94b/bin/serverless.js#L12